File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count=0
        cur=0
        for i in range(0, len(s)):
            if(s[i]==" "):
                cur=0
            else:
                cur+=1
                count=cur         
        return count

File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count=0;
        bool found=false;
        for(int i=s.size()-1; i>=0; i--){
            if(s[i]!=' '){
                found=true;
                count++;
            }else if(found){
                break;
            }
        }
        return count;
    }
};